crossfit/backend/torch/model.py

```diff
@@ -25,16 +26,33 @@
 from crossfit.utils.torch_utils import cleanup_torch_cache, concat_and_pad_tensors
 
 
+class ModelOutputType(Enum):
+    NUMERIC = "numeric"
+    STRING = "string"
+
+
 class Model:
-    def __init__(self, path_or_name: str, max_mem_gb: int = 16, model_output_type: str = "numeric"):
+    def __init__(
+        self,
+        path_or_name: str,
+        max_mem_gb: int = 16,
+        model_output_type: Any = ModelOutputType.NUMERIC,
+    ):
+        """Initialize a Crossfit Pytorch Model Instance.
+
+        Args:
+            path_or_name (str): Path to the model file or the model name to load.
+            max_mem_gb (int, optional): Maximum memory in gigabytes to allocate for the model.
```

Comment:
is it optional? if so signature could reflect that? And also I'd imagine the right data type is float since we could allow for `16.5gb` too

Reply:
This is not optinal yet, it should be, i need to push in a fix to auto detect it . Will do as a followup, will remove this for now.